base_custom_filter/views/ir_custom_filter_views.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+    <record id="view_ir_custom_filter_form" model="ir.ui.view">
+        <field name="name">ir.custom.filter.form</field>
+        <field name="model">ir.custom.filter</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_title">
+                        <h1>
+                            <field name="name" />
+                        </h1>
+                    </div>
+                    <group>
+                        <field name="model_id" options="{'no_create_edit': True}" />
+                        <field name="model_name" invisible="1" />
+                        <field name="group_id" />
```

Comment:
can you pass the model_id in the context here? something like ~ `context: default_model_id: model_id`. This way when creating a filter group on the go, the model will be defaulted.

Reply:
Done! Thank you :)